app/assets/stylesheets/application.css.scss

```diff
@@ -16,76 +16,77 @@
 
  */
 @import 'foundation-icons';
+
+/* CUSTOM VARIABLES */
+
 $boost-red: #B72039;
 
+/* FOUNDATION BREAKPOINTS */
+
+$medium-start: 40.063em;
+$large-start: 63.9375em;
+
+/* BASICS */
+
 body {
+  background-color: $boost-red;
   color: white;
```

Comment:
Could make this a variable

Reply:
Or use $primary-color

Reply:
do you mean make `background-color: $boost-red;` a variable in itself?